docs/phylum.md

```diff
@@ -0,0 +1,37 @@
+---
+title: phylum
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Interact with the Phylum API.
+
+### Core Commands
+* [phylum analyze](https://docs.phylum.io/docs/phylum_analyze)
+* [phylum auth](https://docs.phylum.io/docs/phylum_auth)
+* [phylum projects](https://docs.phylum.io/docs/phylum_projects)
+
+### Additional Commands
+* [phylum help](https://docs.phylum.io/docs/phylum_help)
+* [phylum history](https://docs.phylum.io/docs/phylum_history)
+* [phylum package](https://docs.phylum.io/docs/phylum_package)
+* [phylum ping](https://docs.phylum.io/docs/phylum_ping)
+* [phylum update](https://docs.phylum.io/docs/phylum_update)
+* [phylum version](https://docs.phylum.io/docs/phylum_version)
+
+### Options
+  `-c`, `--config <FILE>`
+  &emsp; Sets a custom configutation file
```

Comment:
```suggestion
  &emsp; Use a specific configuration file
```
Typo...plus it's not really _setting_ a config file so much as it is _using_ a specific one. It is understood that the CLI code will need to change if this full suggestion is accepted...but at least the typo should be fixed here.